### DISCUSSION TITLE
Quit chat!

### DISCUSSION THREAD

**Discussion by om3rx** [Yue-Yang/ChatGPT-Siri#100] [General]
Hi, what is the purpose of quit chat function?
This will just start over and bring the continue-text.. 
Is it possible to quit Siri once Quit Chat is triggered?

**Comment by Yue-Yang** [comment]
Quit chat is to quit the shortcut, so you can use the system siri.

**Reply by suedepup** [reply]
> Quit chat is to quit the shortcut, so you can use the system siri.

Saying "quit chat" is no longer working for me. Chat GPT just keeps on going instead of stopping and closing out. It used to work, but now it doesn't. I tried changing it to "stop chat" but that isn't working either. Any idea why this is happening?
